source/_integrations/alexa.intent.markdown

```diff
@@ -17,17 +17,11 @@ The built-in Alexa integration allows you to integrate Home Assistant into Alexa
 
 ### Requirements
 
-Amazon requires the endpoint of a skill to be hosted via SSL. Self-signed certificates are OK because our skills will only run in development mode. Read more on [our blog][blog-lets-encrypt] about how to set up encryption for Home Assistant. When running Hass.io, using the [Let's Encrypt](/addons/lets_encrypt/) and [Duck DNS](/addons/duckdns/) add-ons is the easiest method. If you are unable to get HTTPS up and running, consider using [this AWS Lambda proxy for Alexa skills](https://community.home-assistant.io/t/aws-lambda-proxy-custom-alexa-skill-when-you-dont-have-https/5230).
+- Amazon Developer Account. You can sign on [here](https://developer.amazon.com).
+- An [AWS account](https://aws.amazon.com/free/) is need if you want to use Alexa Custom Skill API. Part of your Alexa Custom Skill will be hosted on [AWS Lambda](https://aws.amazon.com/lambda/pricing/). However you don't need worry the cost, AWS Lambda allow free to use up to 1 millions requests and 1GB outbound data transfer per month.
+- The Alexa Custom Skill API also needs your Home Assistant instance can be accessed from Internet. We strongly suggest you host HTTPS server and use validation certificate. Read more on [our blog](/blog/2015/12/13/setup-encryption-using-lets-encrypt/) about how to set up encryption for Home Assistant. When running Hass.io, using the [Let's Encrypt](/addons/lets_encrypt/) and [Duck DNS](/addons/duckdns/) add-ons is the easiest method.
```

Comment:
Home Assistant Cloud is the easiest. Second is using the DuckDNS add-on because it has Let's Encrypt built-in.

Reply:
This whole page (and the smart home one) are prefaced by a section introducing Home Assistant Cloud and indicating that these are the "manual" instructions if you choose not to do that, so re-mentioning Home Assistant Cloud may be confusing. I can add another mention of it if you'd prefer though?

Reply:
At least let's just mention the duckdns add-on.